packages/auth/cloud/cloudformation.spec.ts

```diff
@@ -1,3 +1,5 @@
+import { getTemplate } from './cloudformation';
+
 test('test 1', () => {
```

Comment:
Fix the name of your test. You should always think, "How can I name my test to remember the context in five years?"

"test 1" isn't a good name. You could use something like, "getTemplate should return an object."

Reply:
I get it. Fixed it now, Pedro.